tests/Range.test.js

```diff
@@ -79,6 +79,14 @@ describe('Range', () => {
     expect(wrapper.find('.rc-slider-handle').length).toBe(2);
   });
 
+  it('should only update bounds that are out of range', () => {
+    const props = { min: 0, max: 10000, value: [0.01, 10000], onChange: jest.fn() };
+    const range = mount(<Range {...props} />);
```

Comment:
I think here should add step for calc precision, `const range = mount(<Range {...props} step={0.01} />);` , if no step, the precision will be `1`.  Maybe that is why this test case fail before pr.